src/java/org/apache/cassandra/index/sai/memory/TrieMemtableIndex.java

```diff
@@ -362,7 +350,9 @@ public CloseableIterator<PrimaryKeyWithSortKey> orderResultsBy(QueryContext quer
         if (keys.isEmpty())
             return CloseableIterator.emptyIterator();
 
-        if (!orderer.isBM25())
+        if (orderer.isBM25())
+            return orderByBM25(keys.stream(), orderer);
+        else
```

Comment:
Nit: the convention is that if one branch of the conditional needs braces, all do.
```suggestion
        if (orderer.isBM25())
        {
            return orderByBM25(keys.stream(), orderer);
        }
        else
```

Reply:
Can you provide a link where I can read about this convention as it bloats the code? I've looked into Apache Cassandra's code style and haven't found related points.

Reply:
I don't have a link. This is the convention the vector team had when working on SAI. Generally speaking, I'm not sure we need to follow the Apache Cassandra convention on SAI.

Reply:
Actually I realized now that I forgot to remove brackets in the other case.